app/views/admin/articles/new.html.erb

```diff
@@ -1,2 +1,11 @@
-<%= render 'admin/page_header', thing: @article %>
+<% if @collection %>
+  <h1>
+    <%= link_to thing_type(@collection).pluralize, [:admin, thing_type(@collection).pluralize.downcase] %>
```

Comment:
What possible types could this collection ever be? 
Is this future planning or are there multiple types now now?